hack/force-cert-retation.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -xe
+
+KUBECTL=${KUBECTL:-kubectl}
+POD_NAMESPACE=${POD_NAMESPACE:-kubemacpool-system}
+POD_APP_LABEL=${POD_APP_LABEL:-kubemacpool-leader=true}
```

Comment:
ditto

Reply:
This is `POD_LABEL` if we name it `POD_APP_LABEL` can be confused with usual `app` label.